pkgs/by-name/ca/caddy/package.nix

```
@@ -57,7 +57,7 @@ buildGoModule {
       --fish <($out/bin/caddy completion fish) \
       --zsh <($out/bin/caddy completion zsh)
   '';
-
+  tags = [ "nobadger" ];
```

Is there a particular reason why you added `tags` all the way down at line 60, between `postInstall` and `passthru.tests` instead of somewhere at the top, best probably right below `ldflags` but before `nativeBuildInputs`?

If there isn't a reason, please move it up.

It's really confusing and easy to overlook otherwise.

If you feel fancy, you can add a comment why `nobadger` is needed:


```suggestion
  # matches upstream
  tags = [ "nobadger" ];
```

---

Done, thank you!